### PR TITLE
Remove query on placeid

### PR DESCRIPTION
**Pull request by c-w**
As per comment
https://github.com/CatalystCode/project-fortis-pipeline/pull/101/files#r132579693

### CODE REVIEW

File: src/resolvers-cassandra/Messages/queries.js [hunk 1]
```diff
@@ -49,64 +48,61 @@ function byBbox(args, res) { // eslint-disable-line no-unused-vars
     if (!args.bbox || args.bbox.length !== 4) return reject('Invalid bbox specified');
 
     const { fromDate, toDate } = parseFromToDate(args.fromDate, args.toDate);
+    const limit = args.limit || 15;
```
Comment (Smarker) [inline src/resolvers-cassandra/Messages/queries.js:51]: I'd maybe use a name more specific than `limit` to make it more clear why it's needed.

**Review by Smarker — APPROVED** [review]
LGTM

File: src/resolvers-cassandra/Messages/queries.js [hunk 2]
```diff
@@ -49,64 +48,61 @@ function byBbox(args, res) { // eslint-disable-line no-unused-vars
...
+    const [north, west, south, east] = args.bbox;
+
+    const tagsQuery = `
+    SELECT eventid
```
Comment (erikschlegel) [inline src/resolvers-cassandra/Messages/queries.js:55]: would `distinct eventid` work for this query? Trying to mitigate the duplicate record issue.

**Review by erikschlegel — COMMENTED** [review]
Looks good. Just one posted question below.

File: src/resolvers-cassandra/Messages/queries.js (hunk 2 above)
Comment (c-w) [reply src/resolvers-cassandra/Messages/queries.js:55]: @erikschlegel How do you propose to address this?

image

Comment (erikschlegel) [reply src/resolvers-cassandra/Messages/queries.js:55]: it was more of a question which you just answered.

**Review by erikschlegel — COMMENTED** [review]
LGTM

**Pull request merged**